text/0033-view-hierarchy.md

```diff
@@ -38,6 +38,7 @@ The attachment payload is a JSON structure containing the view hierarchy, here's
             "y": 0.0,
             "z": 2.0, // if applies.
             "visible": true|false,
+            "visibility": "visible|invisible|gone", // Android specific visibility property
```

Comment:
Shouldn't this be part of the `{extra_properties}`? since this is not applicable to every platform.

Reply:
I think the idea back then was to have a single flat list for all properties since all of them are optional, can you remember @brustolin?